pjmedia/src/pjmedia-codec/ffmpeg_vid_codecs.c

```diff
@@ -63,7 +59,9 @@
 #endif
 
 #if LIBAVCODEC_VER_AT_LEAST(53,61)
-#  if LIBAVCODEC_VER_AT_LEAST(54,59)
+#  if LIBAVCODEC_VER_AT_LEAST(59,25)
+#      define AVCODEC_HAS_ENCODE(c)	(is_encoder)
```

Comment:
Minor thing, this macro looks rather ugly as `is_encoder` is actually a local var :)
If only init_codec() uses the macros, what if move this AVCODEC_HAS_ENCODE/DECODE #def right-before the function and #undef later (below the function)?